src/tokens/solana.tokenlist.json

```diff
@@ -4068,6 +4068,23 @@
         "coingeckoId": "funfair"
       }
     },
+    {
+      "chainId": 101,
+      "address": "CnGUfvi9FxiRPuaBXpYmaXEwBjj5X6kwNJB2Cba5TiQp",
+      "symbol": "SOLUP",
+      "name": "SolanaUP",
+      "decimals": 8,
+      "logoURI": "https://raw.githubusercontent.com/deeptext2/solup/main/solup-green.svg",
```

Comment:
please store logo in this repo

Reply:
Thank you, We have updated the logo.

Reply:
please add only one token to the list

Reply:
Sure, We add only SOLUP token, Please check the last commit